addons/account/models/account_move.py

```diff
@@ -6798,16 +6798,32 @@ def get_invoice_localisation_fields_required_to_invoice(self, country_id):
     def get_extra_print_items(self):
         """ Helper to dynamically add items in the 'Print' menu of list and form of account.move.
         """
-        if posted_moves := self.filtered(lambda m: m.state == 'posted' and m.is_move_sent):
+        if moves_to_export := self.filtered(lambda m: m.get_move_zip_export_docs()):
```

Comment:
Aren't we basically reverting the fix that was done [here](https://github.com/odoo/odoo/commit/aed7f64a693f7ef7158a4186caae7aee177dc504)?